### PR TITLE
feat: add HTTPS to File Browser Docker

### PR DESCRIPTION
**Pull request by mohamedbenhariz**
**Description**

:rotating_light: Before submitting your PR, please indicate which issues are either fixed or closed by this PR. See [GitHub Help: Closing issues using keywords](https://help.github.com/articles/closing-issues-via-commit-messages/).

- [x] DO make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] DO make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *our master*.
- [x] DO make sure that File Browser can be successfully built. See [builds](https://github.com/filebrowser/community/blob/master/builds.md) and [development](https://github.com/filebrowser/community/blob/master/development.md).
- [x] AVOID breaking the continuous integration build.

**Further comments**

### CODE REVIEW

**Comment by leodrivera** [comment]
This is a long-awaited feature. I hope it gets merged soon

**Comment by zh-pan** [comment]
The feat can use now?

**Comment by mohamedbenhariz** [comment]
> The feat can use now?

Yes you can use it for now , you can clone the repo and use it, in my case i'm using it .

**Comment by zh-pan** [comment]
@mohamedbenhariz Thanks very much!

But I already enable https on `filebrowser:master`  branch now, and this is my config on `docker-compose.yml`:

```yml
services:
  filebrowser:
    image: filebrowser/filebrowser:latest
    container_name: filebrowser
    restart: always
    ports:
      - '8080:80'
    volumes:
      - /path/to/your/srv:/srv
      - /path/to/your/filebrowser.db:/database.db
      - /path/to/your/your.pem:/Certs/your.pem
      - /path/to/your/your.key:/Certs/your.key
    environment:
      - FB_CERT=/Certs/your.pem
      - FB_KEY=/Certs/your.key
```

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,9 @@ RUN chmod +x /healthcheck.sh  # Make the script executable
 HEALTHCHECK --start-period=2s --interval=5s --timeout=3s \
     CMD /healthcheck.sh || exit 1
 
-VOLUME /srv
+RUN mkdir -p /Certs    
+
+VOLUME /srv /Certs
```
Comment (o1egl) [inline Dockerfile]:
```suggestion
RUN mkdir -p /certs    

VOLUME /srv /certs
```

**Comment by leodrivera** [comment]
Is there any prediction of when this PR will be merged?

File: Dockerfile (hunk 1 above)
Comment (mohamedbenhariz) [reply Dockerfile]: Thanks for your Help

**Pull request closed without merging**